dgisim/src/dices.py

```diff
@@ -1,29 +1,62 @@
 from __future__ import annotations
-
 import random
+from collections import Counter
+from functools import lru_cache
+from typing import Any, Optional, Iterator, Iterable
+from typing_extensions import Self, override, TYPE_CHECKING
 from collections import defaultdict
-from typing import Optional, Iterator, Iterable, Union, List, Set, Tuple, Dict
+from functools import cache
+from heapq import heappop, heapify
+
+from typing_extensions import TYPE_CHECKING, Self
 
-from typing_extensions import Self
+if TYPE_CHECKING:
+    from .state.game_state import GameState
+    from .state.player_state import PlayerState
 
-from dgisim.src.element.element import Element
+from dgisim.src.element import Element
 from dgisim.src.helper.hashable_dict import HashableDict
-from dgisim.src.helper.level_print import level_print
-from dgisim.src.helper.quality_of_life import BIG_INT
+from dgisim.src.helper.quality_of_life import BIG_INT, case_val
```

Comment:
all files under the `src/` directory are now using relative import.

in this case they should be

```py
from .element import Element
from .helper.... # continues
```

Reply:
fixed